test/e2e-tests.sh

```diff
@@ -40,6 +40,12 @@
 #   PROJECT_ID:  the GKR project in which to create the new cluster (unless using "--run-tests")
 #   MAX_PARALLEL_TESTS:  The maximum number of go tests to run in parallel (via "-test.parallel", default 12)
 
+# FIXME(ksuszyns): Remove upgrade-tests invocation before merging the PR
+if [[ "$1" == '--consolidated' ]]; then
+  echo 'FIXME(ksuszyns): Remove upgrade-tests invocation before merging the PR'
+  exec "$(dirname "${BASH_SOURCE[0]}")/e2e-upgrade-tests.sh"
```

Comment:
do we want this to be it's own job?
or part of the `consolidated` ? 

We have already separated jobs

Reply:
This is just temporal. To showcase that it actually works. See the build: https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative-sandbox_eventing-kafka/467/pull-knative-sandbox-eventing-kafka-integration-test-channel-consolidated/1377556813607604224

knative/test-infra#2735 is to add a separate presubmit job

Reply:
ok